dipy/denoise/gibbs.py

```diff
@@ -220,7 +223,7 @@ def _gibbs_removal_2d(image, n_points=3, G0=None, G1=None):
     return imagec
 
 
-def gibbs_removal(vol, slice_axis=2, n_points=3, inplace=True):
+def gibbs_removal(vol, slice_axis=2, n_points=3, inplace=True, num_processes=1):
```

Comment:
Please use `num_threads` instead for consistency with other multi-threaded dipy functions such as (e.g. `dipy.denoise.nlmeans_3d`, functions in `dipy.align.streamlinear, etc.)

Reply:
The default of 1 here looks good to me as this is compatible with users potentially applying other means of parallelization to call dipy functions (e.g. joblib, etc).  I did see that some functions in dipy do default to `num_threads=None`, though. Also, it looks like other functions use `num_threads=None` rather than `num_threads=0` to indicate the maximum should be used so it is probably best to do the same here.

Reply:
Changed to `num_threads` with `None` being use all cores.